java/src/jmri/jmrit/logixng/LogixNGPreferences.java

```diff
@@ -93,4 +93,16 @@ public interface LogixNGPreferences {
      */
     public boolean getTreeEditorHighlightRow();
 
+    /**
+     * Set whether system names should be shown or not.
+     * @param value true if the row should be highlighted, false not
```

Comment:
Description does not match.

Reply:
Thanks. I cannot resolve this now but I can fix it in a separate PR.